core/templates/pages/topics-and-skills-dashboard-page/topics-and-skills-dashboard-page.directive.html

```diff
@@ -358,15 +358,15 @@
   }
   .oppia-topics-and-skills-dashboard .dashboard-title {
     color: #01645c;
-    font-family: 'Capriola', 'Roboto', Arial, sans-serif;
+    font-family: Capriola, Roboto, Arial, sans-serif;
     font-size: 3em;
     margin-bottom: 25px;
     margin-top: 0;
     padding-top: 78.5px;
     text-align: center;
   }
   .oppia-topics-and-skills-dashboard .intro-card {
-    background: #fff;
+    background: #ffffff;
```

Comment:
I'm still seeing  `#fff` in L378 and `#eee` in L389.

Reply:
Done.